src/config/dynamic-theme-fixes.config

```diff
@@ -13713,6 +13713,19 @@ a[class^="PaginationButton"]:hover {
     background-color: var(--darkreader-bg--nyh-color-white);
 }
 
+
+================================
+
+sw.kovidgoyal.net
+
+CSS
+.sidebar-drawer, .sidebar-search {
+  background: var(--darkreader-neutral-background) !important;
+}
+.sidebar-tree a.current, .sidebar-tree label:hover {
+  background: var(--darkreader-selection-background) !important;
+}
```

Comment:
What is the purpose of these 2 selectors?

Reply:
it's for the background of the selected sidebar item and the ⌃/⌄arrows for expand/collapse when hovering over them